airgun/entities/host_new.py

```diff
@@ -399,43 +399,6 @@ def get_tracer(self, entity_name):
         self.browser.plugin.ensure_page_safe()
         return view.traces.read()
 
-    def get_os_info(self, entity_name):
-        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
-        view.wait_displayed()
-        self.browser.plugin.ensure_page_safe()
-        return view.details.operating_system.read()
-
-    def get_provisioning_info(self, entity_name):
-        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
-        view.wait_displayed()
-        self.browser.plugin.ensure_page_safe()
-        return view.details.provisioning.read()
-
-    def get_bios_info(self, entity_name):
-        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
-        view.wait_displayed()
-        self.browser.plugin.ensure_page_safe()
-        return view.details.bios.read()
-
-    def get_registration_details(self, entity_name):
-        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
-        view.wait_displayed()
-        self.browser.plugin.ensure_page_safe()
-        return view.details.registration_details.read()
-
-    def get_hw_properties(self, entity_name):
-        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
-        view.wait_displayed()
-        self.browser.plugin.ensure_page_safe()
-        return view.details.hw_properties.read()
-
-    def get_provisioning_templates(self, entity_name):
-        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
-        view.wait_displayed()
-        self.browser.plugin.ensure_page_safe()
-        d = view.details.read()
-        return d['provisioning_templates']['templates_table']
-
```

Comment:
all these methods are been used as part of `HostDetailsCard`?

Reply:
I think we need to also open a parallel PR in robottelo to remove these function usages as well! Do we have any, if yes please link here

Reply:
@LadislavVasina1 what's your view here? are these functions are necessary?

Reply:
@omkarkhatavkar I wrote those entities to be used by the user as an oneliner that could help him/her simplify the development of UI tests in a way that they don't have to go through the structure of dictionary returned by `view.details.read()` and look for specific keys that they would need to achieve desired output. In my opinion they could stay there as they can probably be useful to someone in future.

Reply:
@sambible based on  @LadislavVasina1 comments, can you please keep these functions? and revert it.